storage/migrations/5_windowpost.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 5 provides additional tracking of the miner->deadline->partition->sector relation
+
+func init() {
+
+	up := batch(`
+	CREATE TABLE IF NOT EXISTS "miner_sector_posts" (
+		"miner_id" text not null,
+		"sector_id" int not null,
```

Comment:
bigserial is autoincrementing

Reply:
I can't explain why we have it in those three tables :cry:

Reply:
It's because go-pg uses bigserial when a bigint is part of a primary key. See https://github.com/go-pg/pg/blob/v10.3.1/orm/table_create.go#L144 and the `pkSQLType` function

Reply:
`bigserial` is basically an alias for `bigint` with a hidden sequence. You can insert into it normally but it will also take the next increment if you specify DEFAULT as its value. Annoying that we have it but not a disaster. Perhaps we should add a migration to bigint in this PR so they are all consistent.

Reply:
Tracking here: https://github.com/filecoin-project/sentinel-visor/issues/107